### PR TITLE
Update the README with Shim Info

### PR DESCRIPTION
**Pull request by JacksonWeber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,51 @@ process.env.APPLICATIONINSIGHTS_CONFIGURATION_FILE = "C:/applicationinsights/con
...
+
+And invoked via `appInsights.<METHOD_NAME>`
+|Property                     |Support Status                                                                                              |
+| ----------------------------|------------------------------------------------------------------------------------------------------------|
+| setDistributedTracingMode   | AI only tracing mode is no longer supported. Migrate to using W3C_AND_AI tracing mode. |
+| setAutoCollectHeartbeat     | Heartbeat is not supported in either the distro or the shim.|
+| enableWebInstrumenatation   | WebInstrumentation is not supported in either the distro or the shim. |
+| setAutoDependencyCorrelation| Turning off autoDependencyCorrelation is not supported by either the distro or the shim. |
+| setUseDiskRetryCaching      | While enabling/disabling offline storage is supported, setting the resend interval or the maxBytesOnDisk values are not supported in the shim or the distro. |
+| setAutoCollectIncomingRequestAzureFunctions | Auto collection of Azure Functions is not supported by the shim. Please migrate to the distro for support. |
```
Comment (hectorhdzg) [inline README.md:179]: Distro doesn't support this either

File: README.md [hunk 2]
```diff
@@ -157,6 +157,51 @@ process.env.APPLICATIONINSIGHTS_CONFIGURATION_FILE = "C:/applicationinsights/con
...
+| enableAutoCollectIncomingRequestAzureFunctions | Not supported by the shim. Migrate to the distro to use auto collection of Azure Functions. |
+| enableUseAsyncHooks | Using async hooks is not supported by the shim or the distro as it is not supported by OpenTelemetry. |
+| enableResendInterval | Not supported by the shim. It is possible to configure the interval between exports via OpenTelemetry span processors, but not specifically cached events. The @azure/monitor-opentelemetry-exporter uses a resend interval of one minute. |
+| enableMaxBytesOnDisk | Not supported by the shim. And not supported to be changed in the distro. The @azure/monitor-opentelemetry-exporter sets this value at 50MB. |
+| noHttpAgentKeepAlive | Not supported in the shim or the distro. |
+| httpAgent/httpsAgent | Not supported in the shim or the distro. |
+| enableWebInstrumentation | Not currently supported in the shim or the distro. |
+| webInstrumentationConnectionString | Not supported in the shim or the distro. |
+| webInstrumentationConfig | Not currently supported by the shim or the distro. |
+
```
Comment (hectorhdzg) [inline README.md:203]: We are still lacking TelemetryClient documentation,, if you are planning to follow up later please create a task so we do not forget

File: README.md [hunk 3]
```diff
@@ -157,6 +157,51 @@ process.env.APPLICATIONINSIGHTS_CONFIGURATION_FILE = "C:/applicationinsights/con
...
+|-----------------------|-----------------------------------------------------------------------|
+| instrumentationKey & endpointUrl | Not supported by the shim or the distro. Please migrate to using the connectionString. |
+| maxBatchSize | Not supported by the shim but can be configured by using OpenTelemetry SpanProcessors in the distro. |
+| disableAppInsights | Not supported by the shim. Disabling telemetry export is possible via the distro using OpenTelemetry.|
+| correlationIdRetryIntervalMs | Not supported by either the shim or the distro as the value is not configurable in OpenTelemetry.|
+| ignoreLegacyHeaders | Legacy headers in outgoing requests are not supported in the shim or the distro. Therefore they will always be disabled. |
+| distributedTracingMode | Distributed tracing mode is always set to AI_AND_W3C. AI only tracing mode is not supported in the shim or the distro.|
+| enableLoggerErrorToTrace | Not supported in the shim or the distro as all errors will be logged as exceptions in both. |
+| enableAutoCollectHeartbeat | Not supported in the shim or the distro. |
```
Comment (hectorhdzg) [inline README.md:192]: Same here, deprecated functionality

File: README.md (hunk 2 above)
Comment (JacksonWeber) [reply README.md:203]: Added unsupported TelemetryClient fields here, as well as noting that the required fields for `trackPageView` are slightly different.

**Review by hectorhdzg — APPROVED** [review]
LGTM

**Pull request merged**